<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Buttongroup" name="title_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>50</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="mydisk">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 255);
font: 10pt &quot;宋体&quot;;</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="mydiskwg" name="page_mydisk">
           <attribute name="title">
            <string>全部文件</string>
           </attribute>
          </widget>
          <widget class="mypicwg" name="page_mydoc">
           <attribute name="title">
            <string>我的文档</string>
           </attribute>
          </widget>
          <widget class="mypicwg" name="page_mypics">
           <attribute name="title">
            <string>我的图片</string>
           </attribute>
          </widget>
          <widget class="mypicwg" name="page_mysound">
           <attribute name="title">
            <string>我的音频</string>
           </attribute>
          </widget>
          <widget class="mypicwg" name="page_myvideo">
           <attribute name="title">
            <string>我的视频</string>
           </attribute>
          </widget>
          <widget class="mypicwg" name="page_other">
           <attribute name="title">
            <string>其他文件</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="transferwg" name="page_transfer_record"/>
      <widget class="sharefilelist" name="page_sharelist"/>
      <widget class="ranklist" name="page_downloadlist"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Buttongroup</class>
   <extends>QWidget</extends>
   <header location="global">buttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mydiskwg</class>
   <extends>QWidget</extends>
   <header location="global">mydiskwg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>transferwg</class>
   <extends>QWidget</extends>
   <header location="global">transferwg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mypicwg</class>
   <extends>QWidget</extends>
   <header location="global">mypicwg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>sharefilelist</class>
   <extends>QWidget</extends>
   <header location="global">sharefilelist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ranklist</class>
   <extends>QWidget</extends>
   <header location="global">ranklist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
